python-threatexchange/threatexchange/cli/tests/sample_data_e2e_test.py

```diff
@@ -13,7 +13,7 @@ class SampleDataE2ETest(ThreatExchangeCLIE2eTest):
     def test_direct_to_match(self):
         """The classic first use of the CLI"""
         self.assert_cli_output(
-            ("match", "text", "-I", self.MATCHES_ONE),
+            ("match", "text", "--", self.MATCHES_ONE),
```

Comment:
Do we not want to keep "-I" for inline?

Reply:
The functionality is still there, just renamed as `--`!